Gemfile

```diff
@@ -37,9 +38,14 @@ if RUBY_VERSION < '2.0.0'
   gem 'mime-types', '< 3'
 end
 
+
 # Capybara versions that support RSpec 3 only support RUBY_VERSION >= 1.9.3
 if RUBY_VERSION >= '1.9.3'
-  gem 'capybara', '~> 2.2.0', :require => false
+  if /5(\.|-)1/ === RAILS_VERSION || "master" == RAILS_VERSION
+    gem 'capybara', '~> 2.13', :require => false
+  else
+    gem 'capybara', '~> 2.2.0', :require => false
```

Comment:
The pin to `2.2.0` prevents us from bumping further than the highest `2.2.x` rails needs at least `2.13`. However, we pin to the earliest version that we support, so this is why this is conditionaled like this.